### DISCUSSION TITLE
Dubbi su integrazione Bticino Smart

### DISCUSSION THREAD

**Discussion by Matty1184** [indomus/forum#2115] [GENERALE - Discussioni generiche]
Ciao a tutti, chiedo il vostro aiuto in quanto ho diversi dubbi e confusioni nella mia testa sull'integrazione di alcuni dispositivi.
Ho HA su Raspberry con Sonoff Zigbee 3.0 USB Dongle Plus e impianto Bticino Smart con il suo Gateway che attualmente funziona tramite sua app e suo Gateway.
Qual è la strada migliore da percorrere? vorrei che il tutto fosse gestito da HA ma leggendo varie guide alcuni consigliano di eliminare il gateway e utilizzare antenna Sonoff, altri di utilizzare l'integrazione Netnamo.
ringrazio tutti in anticipo!

**Comment by zuccospooky** [comment]
ciao, sono anche io curioso perchè mi sto avvicinando a una soluzione simile. Hai per caso testato i due sistemi e hai trovato delle differenze?

**Reply by Matty1184** [reply]
Ciao, al momento ho integrato il gateway tramite homekit e mi funziona tutto correttamente. avevo provato a associare alcuni switch direttamente tramite zigbee ma continuavano a perdermi il segnale, onestamente stavo sbagliando sicuramente qualcosa ma non ho capito esattamente cosa!

**Comment by mefrasa** [comment]
Io ho sfruttato l'integrazione HomeKit (che penso sia la strada più semplice).
Questa strada ha il vantaggio di lasciare la App nativa, completamente funzionante, ma si perde la possibilità di utilizzare alcuni componenti (per es. il misuratore di consumi)
